pkg/v22/controllercontext/status.go

```diff
@@ -5,5 +5,5 @@ type ContextStatus struct {
 }
 
 type ContextStatusWorker struct {
-	Nodes int32
+	Nodes int
```

Comment:
I based this on the clusterapi resource but its simpler to just use an int. It will be converted to YAML anyway.